### PR TITLE
feat: Add coins from slack

### PR DESCRIPTION
**Pull request by cgkol**
https://linear.app/uniswap/issue/DAT-260/default-token-list-addition-request-ondo-finance
https://linear.app/uniswap/issue/DAT-261/default-token-list-addition-request-velodrome-finance
https://linear.app/uniswap/issue/DAT-262/default-token-list-addition-request-aerodrome-finance

### CODE REVIEW

File: src/tokens/optimism.json [hunk 1]
```diff
@@ -22,5 +22,13 @@
     "symbol": "USDC.e",
     "decimals": 6,
     "logoURI": "https://ethereum-optimism.github.io/data/USDC/logo.png"
+  },
+  {
+    "chainId": 10,
+    "address": "0x9560e827af36c94d2ac33a39bce1fe78631088db",
```
Comment (DefiPanda) [inline src/tokens/optimism.json:28]: checksum this?

Comment (cgkol) [reply src/tokens/optimism.json]: i juuuust pushed a new PR

**Review by DefiPanda — APPROVED** [review]
Shall we also bump up the package version?

File: src/tokens/optimism.json (hunk 1 above)
Comment (DefiPanda) [reply src/tokens/optimism.json]: Ah nice

**Pull request merged**